docs/develop/authorize-to-microsoft-graph.md

```diff
@@ -59,3 +59,16 @@ For examples of detailed walkthroughs and scenarios, see:
 * [Create a Node.js Office Add-in that uses single sign-on](create-sso-office-add-ins-nodejs.md)
```

Comment:
Remember to update ms.date in the various files

Reply:
good catch